addon/synthDrivers/dengjen_neural_voices/grpc_client/__init__.py

```
@@ -112,7 +116,7 @@ def start_grpc_server():
     try:
         server_log_file = os.path.join(DENGJEN_VOICES_BASE_DIR, "logs", "sonata-grpc.log")
         Path(server_log_file).parent.mkdir(parents=True, exist_ok=True)
-        server_stdout = open(server_log_file, "wb")
+        server_stdout = SERVER_LOG_HANDLE = open(server_log_file, "wb")
```

Ruff (0.16.2)

[warning] Use a context manager for opening files

(SIM115)